cardano-db/src/Cardano/Db/Error.hs

```diff
@@ -34,6 +34,7 @@ data LookupFail
   | DBMultipleGenesis
   | DBExtraMigration !String
   | DBPruneConsumed !String
+  | DBResetJsonb !String
```

Comment:
I know you didn't create this, but why not Text here?

Reply:
it's because `SqlError` has a `Show` instance and we can just concat the `SqlError` and `LookupFail` without converting `Text -> String`.